kew.js

```diff
@@ -38,6 +38,40 @@
 			for (var i = 0; i < this.length; i++) {
 				this.elements[i].parentElement.removeChild(this.elements[i]);
 			}
+		},
+		add: function(htmlString) {
+			if (htmlString == "") {
+				throw new Error("Parameter in .add must not be an empty string.");
+			}
+
+			for (var i = 0; i < this.length; i++) {
```

Comment:
Oh, I see. Thanks! Never thought of that. Any explanations why it'll be creating a **memory leak** if I kept reusing the same variable?